src/server/scripts/Shadowlands/zone_exiles_reach.cpp

```diff
@@ -0,0 +1,388 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "Object.h"
+#include "Player.h"
+#include "ScriptMgr.h"
+#include "Transport.h"
+#include "Conversation.h"
+#include "TemporarySummon.h"
+
+// Scripting in this section is from login to arriving on beach for alliance and horde
+
+enum QuestScriptQuestWarmingUp
+{
+    QUEST_WARMING_UP                = 59926,
+    CONVERSATION_WARMING_UP         = 12798,
+    NPC_WARLORD_BREKA_GRIMAXE1      = 166573,
+    NPC_WARLORD_BREKA_GRIMAXE2      = 166824
+};
+
+class quest_warming_up : public QuestScript
+{
+public:
+    quest_warming_up() : QuestScript("quest_warming_up") { }
+
+    void OnQuestStatusChange(Player* player, Quest const* /*quest*/, QuestStatus /*oldStatus*/ , QuestStatus newStatus) override
+    {
+        switch (newStatus)
+        {
+            case QUEST_STATE_COMPLETE:
+                Conversation::CreateConversation(CONVERSATION_WARMING_UP, player, *player, player->GetGUID(), nullptr);
+                break;
+            case QUEST_STATUS_REWARDED:
+                if (Creature* breka1 = player->FindNearestCreature(NPC_WARLORD_BREKA_GRIMAXE1, 5.0f))
+                    if (Creature* breka2 = player->FindNearestCreature(NPC_WARLORD_BREKA_GRIMAXE2, 75.0f))
+                        breka2->SummonPersonalClone(breka1->GetPosition(), TempSummonType(TEMPSUMMON_TIMED_DESPAWN), 18s, 0, 0, player);
```

Comment:
Remove the 18s timer, its wrong. It should despawn when it reaches its destination. You gotta despawn it and destination reach.

Reply:
its despawned via sai, waypoint end